static/app/views/app/index.tsx

```diff
@@ -38,18 +38,27 @@ function App({children}: Props) {
   const config = useLegacyStore(ConfigStore);
 
   // Command palette global-shortcut
-  useHotkeys('command+shift+p, command+k, ctrl+shift+p, ctrl+k', e => {
-    openCommandPalette();
-    e.preventDefault();
-  });
+  useHotkeys([
+    {
+      match: 'command+shift+p, command+k, ctrl+shift+p, ctrl+k',
+      callback: e => {
+        openCommandPalette();
+        e.preventDefault();
+      },
+    },
+  ]);
 
   // Theme toggle global shortcut
   useHotkeys(
-    'command+shift+l, ctrl+shift+l',
-    e => {
-      ConfigStore.set('theme', config.theme === 'light' ? 'dark' : 'light');
-      e.preventDefault();
-    },
+    [
```

Comment:
Why does it take an array?

Reply:
So we can pass multiple hotkey combinations and their callbacks instead of having to hard-define each one as its own hook in the final searchbar's implementation